embc-app/Services/Registrations/NewRegistrationService.cs

```diff
@@ -42,7 +42,7 @@ private EmailMessage CreateEmailMessageForRegistration(ViewModels.Registration r
         {
             var essRegistrationLink = @"<a target='_blank' href='https://justice.gov.bc.ca/embcess/self-registration'>Evacuee Self-Registration</a>";
             var emergencyInfoBCLink = @"<a target='_blank' href='https://www.emergencyinfobc.gov.bc.ca/'>Emergency Info BC</a>";
-
+            var surveyLink = @"<a href='https://pbsurvey.jag.gov.bc.ca/Survey5/' target='_blank'>CLICK HERE</a>";
```

Comment:
is the url 'stable'?

Reply:
As per Ann Marie, the url is stable. If they change it next week we will PR a new change. She doesn't want us to turn this into a env var, so hard-coding it is fine for now as per business needs.